src/dso_api/dynamic_api/permissions.py

```diff
@@ -119,70 +119,90 @@ def has_permission_for_models(self, request, view, models):
         log_access(request, access)
         return access
 
-    def _filters_ok(self, request, schema: DatasetTableSchema) -> bool:  # noqa: C901
-        """Check field authorization for requested filters."""
 
-        # Workaround for BBGA, which has a broken schema and is queried in a funny way
-        # by another application.
-        if schema.dataset.id == "bbga":
-            return True
+def _filters_ok(request, schema: DatasetTableSchema) -> bool:  # noqa: C901
+    """Check field authorization for requested filters."""
 
-        # Type hack: DRF requests have a __getattr__ that delegates to an HttpRequest.
-        request = cast(HttpRequest, request)
+    # Workaround for BBGA, which has a broken schema and is queried in a funny way
+    # by another application.
+    if schema.dataset.id == "bbga":
+        return True
 
-        # And then we and authorization_django monkey-patch the scopes onto the request.
-        scopes = cast(UserScopes, request.user_scopes)
+    # Type hack: DRF requests have a __getattr__ that delegates to an HttpRequest.
+    request = cast(HttpRequest, request)
 
-        # Collect mandatory filters. We need to make an exception for these to handle cases
-        # where the field they reference isn't otherwise accessible.
-        # The actual check for whether the filter sets are provided lives elsewhere.
-        mandatory = set()
-        for profile in scopes.get_active_profile_tables(schema.dataset.id, schema.id):
-            for f in profile.mandatory_filtersets:
-                mandatory.update(f)
+    # And then we and authorization_django monkey-patch the scopes onto the request.
+    scopes = cast(UserScopes, request.user_scopes)
 
-        for key, values in request.GET.lists():
-            if key in ("fields", "format", "page", "page_size", "sorteer") or key.startswith("_"):
-                continue
+    # Collect mandatory filters. We need to make an exception for these to handle cases
+    # where the field they reference isn't otherwise accessible.
+    # The actual check for whether the filter sets are provided lives elsewhere.
+    mandatory = set()
+    for profile in scopes.get_active_profile_tables(schema.dataset.id, schema.id):
+        for f in profile.mandatory_filtersets:
+            mandatory.update(f)
```

Comment:
mandatory.update(*profile.mandatory_filtersets)

Reply:
Should I have indented it? Not sure how this works.

Reply:
Dit werkt niet. Je moet echt <kbd>Ctrl</kbd>+<kbd>G</kbd> doen :)

Reply:
Use the leftmost button to the right of "Write" and "Preview" in the comment edit box.